app/templates/components/programyear-list.hbs

```diff
@@ -74,17 +77,30 @@
 
                 <td class='text-center'>{{publication-status item=programYear showIcon=false}}</td>
                 <td class='text-center'>
-                  {{#action-menu}}
-                    <li {{action 'edit'}}>Edit</li>
-                  {{/action-menu}}
+                  {{#link-to 'programYear.index' program programYear class="program-year-link"}}{{fa-icon 'edit'}}{{/link-to}}
+                  <span class="clickable trash {{if programYear.showRemoveConfirmation 'confirm-removal'}}" {{action 'remove' programYear}}>{{fa-icon 'trash'}}</span>
                 </td>
               </tr>
+
+              {{#if programYear.showRemoveConfirmation}}
+                <tr class='confirm-removal'>
+                  <td colspan=8>
+                    <div class='confirm-message'>
+                      {{t 'programs.confirmRemove' courseCount=program.courseCount}} <br />
```

Comment:
I think this is the wrong removal confirmation.  Shouldn't it be programYears.confirmRemove?